src/Integration/NewConnectedMode/ConfigurationProvider.cs

```diff
@@ -19,19 +19,34 @@
  */
 
 using System;
+using System.ComponentModel.Composition;
 using System.IO;
+using Microsoft.VisualStudio.Shell;
 using SonarLint.VisualStudio.Core.Binding;
 using SonarLint.VisualStudio.Integration.Persistence;
 
 namespace SonarLint.VisualStudio.Integration.NewConnectedMode
 {
+    [Export(typeof(Core.Binding.IConfigurationProvider))]
```

Comment:
The relevant change

Reply:
It's very easy to miss that the export is of interface X whereas the class implements interface Y because the interface names are the same.

I don't think we should try to hide this; it will just cause problems later.